elasticsearch_collectd.py

```diff
@@ -95,17 +95,17 @@
     "cluster.relocating-shards",
     "thread_pool.rejected",
     "jvm.uptime",
-    "indices.total.filter-cache.memory-size",
-}
+    "indices.total.filter-cache.memory-size"
+])
 
-DEFAULTS.update({
+DEFAULTS.update([
     # ADD ADDITIONAL METRIC NAMES
     # TO INCLUDE BY DEFAULT
     "cluster.status",
     "indices.indexing.index-time",
     "indices.merges.time"
```

Comment:
I have addressed this comment and squashed the commit.  I'm not sure why github isn't hiding the comment.